src/components/shared/BreadCrumb/SingleCrumb.tsx

```diff
@@ -1,19 +1,18 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 
 import NavLink from '~shared/NavLink';
 
-import { CrumbText } from './BreadCrumb';
-
 import styles from './BreadCrumb.css';
 
 interface Props {
-  crumbText: CrumbText;
+  crumbText: string | ReactElement;
```

Comment:
So this will no longer accept Message Descriptors ? 

I think it should still accept them

Reply:
Done